src/services/AuthorizationRequestProcessor.ts

```diff
@@ -62,11 +62,29 @@ export class AuthorizationRequestProcessor {
 
 			this.metrics.addMetric("found session", MetricUnits.Count, 1);
 
-			if (session.authSessionState !== AuthSessionState.CIC_DATA_RECEIVED) {
-				this.logger.error(`Session is in the wrong state: ${session.authSessionState}, expected state should be ${AuthSessionState.CIC_DATA_RECEIVED}`, { 
-					messageCode: MessageCodes.INCORRECT_SESSION_STATE,
-				});
-				return new Response(HttpCodesEnum.UNAUTHORIZED, `Session is in the wrong state: ${session.authSessionState}`);
+			switch (session.authSessionState) {
+			  case AuthSessionState.CIC_SESSION_CREATED:
```

Comment:
Is this the same as the default case and if so do we need to explicitly add it?

Reply:
Nah so this state means /claimedIdentity hasn't been called yet and so this is a legit error, we're calling /session then /auth when it should be /session then /claimedIdentity then /auth

Reply:
I see what you mean now, will combine them